.github/workflows/pypi-publish.yml

```diff
@@ -1,10 +1,11 @@
 # -*- coding: utf-8 -*-
 #
-# This file is part of Invenio.
-# Copyright (C) 2020 CERN.
+# Copyright (C) 2021-2025 CERN.
+# Copyright (C) 2025 KTH Royal Institute of Technology.
+# Copyright (C) 2024 Graz University of Technology.
 #
-# Invenio is free software; you can redistribute it and/or modify it
-# under the terms of the MIT License; see LICENSE file for more details
+# Invenio App RDM is free software; you can redistribute it and/or modify
```

Comment:
i would keep invenio only, since the flask packages are used not only with InvenioRDM or invenio-app-rdm